content/en/news/releases/1.17.x/announcing-1.17/change-notes/index.md

```diff
@@ -0,0 +1,144 @@
+  ([Issue #42212](https://github.com/istio/istio/issues/42212)) [PR](https://github.com/istio/istio/pull/42230)
+
+- **Fixed** an issue where the sync timeout setting doesn't work on the remote clusters. [PR](https://github.com/istio/istio/pull/42252)
+
+- **Fixed** Kubernetes service `exportTo` annotation not working on gateways by fixing gateway service dependencies. [Issue #42400]( https://github.com/istio/istio/issues/42400) [PR](https://github.com/istio/istio/pull/42422)
+
+- **Fixed** locality label missing for a sidecar without service selected. [PR](https://github.com/istio/istio/pull/42412)
+
+- **Fixed** an issue where the network endpoints are incorrectly computed when network gateway changes. [Issue #42818](https://github.com/istio/istio/issues/42818) [PR](https://github.com/istio/istio/pull/42825)
+
+- **Fixed** auto-passthrough gateways not getting XDS pushes on service updates if `PILOT_FILTER_GATEWAY_CLUSTER_CONFIG` is enabled. [PR](https://github.com/istio/istio/pull/42721)
+
+- **Fixed** VirtualService delegate behavior not working with `defaultVirtualServiceExportTo: ["."]` setting. [Issue #42602](https://github.com/istio/istio/issues/42602) [PR](https://github.com/istio/istio/pull/42604)
+
+- **Fixed** Pilot push XDS panic when `PortLevelSettings[].Port` is nil leading to abnormal exit of Pilot. [Issue #42598](https://github.com/istio/istio/issues/42598) [PR](https://github.com/istio/istio/pull/42599)
+
+- **Fixed** a bug that caused the Namespace's network label to have a higher priority than the Pod's network label. [Issue #42675](https://github.com/istio/istio/issues/42675) [PR](https://github.com/istio/istio/pull/42676)
+
+## Security
+
+- **Added** validation warning message for L7 Deny rules which will block all TCP traffic under the scope of the policy having that rule. [Issue #25832] [PR](https://github.com/istio/istio/pull/41802)
+
+- **Added** support for using QAT (`QuickAssist Technology`) `PrivateKeyProvider` in SDS. [PR](https://github.com/istio/istio/pull/42203)
+
+- **Added** configuration for selecting QAT private key provider for gateways and sidecars. [PR](https://github.com/istio/api/pull/2565)
+
+- **Added** support to Copy JWT claims to HTTP request headers. [Issue #39724](https://github.com/istio/istio/issues/39724) [PR](https://github.com/istio/api/pull/2570)
+
+- **Fixed** an issue preventing istio-proxy to access root CA when `automountServiceAccountToken` is `false` and `PILOT_CERT_PROVIDER` is `kubernetes`. [PR](https://github.com/istio/istio/pull/42233)
+
+## Telemetry
+
+- **Updated** the Telemetry API to use a new native extension(stats) for Prometheus stats instead of the Wasm-based extension. This improves CPU overhead and memory usage of the feature. Custom dimensions no longer require regex and bootstrap annotations. If customizations use CEL expressions with Wasm attributes, they are likely to be affected. [PR](https://github.com/istio/istio/pull/41441)
```

Comment:
```suggestion
- **Updated** the Telemetry API to use a new native extension (stats) for Prometheus stats instead of the Wasm-based extension. This improves CPU overhead and memory usage of the feature. Custom dimensions no longer require regex and bootstrap annotations. If customizations use CEL expressions with Wasm attributes, they are likely to be affected. [PR](https://github.com/istio/istio/pull/41441)
```